.gitignore

```diff
@@ -1,6 +1,9 @@
 *.swp
 *.beam
+*.bk
```

Comment:
can you explain what .bk is? I didn't find such files in my rust repos, but I may not be looking hard enough and/or some rust repos may have different requirements than others